docs/js/react.js

```diff
@@ -7364,6 +7364,7 @@ var ReactDOM = mapObject({
   ellipse: 'ellipse',
   g: 'g',
   line: 'line',
+  image: 'image',
```

Comment:
It looks like you reverted the wrong thing. The changes in `src/` are appropriate. The change in `docs/` (the file we're commenting in right now) is not.